src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -69,6 +81,24 @@ export default function withAnnotations<P extends object>(
             this.state = { ...defaultState, activeAnnotationId };
         }
 
+        componentDidUpdate(
+            _prevProps: WithAnnotationsProps,
+            { error: prevError, hasAnnotatorInit: prevHasAnnotatorInit }: AnnotatorState,
+        ) {
+            const { error, hasAnnotatorInit } = this.state;
+
+            const hasAnnotatorChanged = prevHasAnnotatorInit !== hasAnnotatorInit;
+            const hasErrorChanged = prevError !== error;
+
+            if ((hasAnnotatorChanged || hasErrorChanged) && error) {
```

Comment:
Should this be `(hasAnnotatorChanged || hasErrorChanged) && error && hasAnnotatorInit`?